src/orion/service/broker/local.py

```diff
@@ -0,0 +1,198 @@
+"""Simple broker that recreates the ExperimentClient on every request"""
+import datetime
+import logging
+from typing import Dict
+
+from bson import ObjectId
+
+from orion.core.utils.exceptions import BrokenExperiment
+from orion.service.broker.broker import (
+    ExperimentBroker,
+    RequestContext,
+    create_experiment_client,
+    get_storage_for_user,
+    retrieve_experiment_client,
+    success,
+)
+from orion.service.client.base import RemoteTrial
+
+log = logging.getLogger(__name__)
+
+
+class LocalExperimentBroker(ExperimentBroker):
+    """Creates an experiment client for each request and process the request.
+    Parallel requests will instantiate different clients which might generate race conditions,
+    locks are required.
+
+    Everything keeps being reinstantiated.
+
+    This is easy to scale because nothing is kept between request,
+    we can just load balance the request through n servers.
+
+    """
+
+    def new_experiment(self, request: RequestContext) -> Dict:
+        log.debug("Spawning new experiment")
+
+        client = create_experiment_client(request)
+        # pylint: disable=protected-access
+        euid = str(client._experiment.id)
+
+        # pylint: disable=protected-access
+        return success(
+            dict(
+                experiment_name=str(client.name),
+                euid=euid,
+                space=client._experiment.space.configuration,
+                version=client._experiment.version,
+                mode="x",
+                max_trials=client._experiment.max_trials,
+                working_dir=client._experiment.working_dir,
+                metadata=client._experiment.metadata,
+            )
+        )
+
+    def suggest(self, request: RequestContext) -> Dict:
```

Comment:
This should handle the kind of exceptions than may be raised by suggest: https://github.com/Epistimio/orion/blob/develop/src/orion/client/experiment.py#L550-L565

Reply:
They are handled at the top level by the [server](https://github.com/Epistimio/orion/pull/1023/files#diff-f6dfba39c2c9777888fc02f84c85109cdeedc4b952a29a4e0c3c2a46f6ef4178R90)